src/identify/index.js

```diff
@@ -175,6 +175,7 @@ class IdentifyService {
     // Update peers data in PeerStore
     this.peerStore.addressBook.set(id, listenAddrs.map((addr) => multiaddr(addr)))
     this.peerStore.protoBook.set(id, protocols)
+    this.peerStore.metadataBook.set(id, 'AgentVersion', Buffer.from(message.agentVersion))
```

Comment:
I was thinking that `metadataBook` would only be used by the application layer.
I think it is useful to store the agentVersion, but I wonder if we should have a way of identifying who stored data in the metadataBook

Reply:
If there is a need to separate them in the future we could make an internal book for libp2p but I'd prefer to avoid that unless a problem arises. Worst case scenario users/apps can override values, but if we document what libp2p stores there I think it should be fine.